### PR TITLE
prevent too much mint

### PR DESCRIPTION
**Pull request by sasurobert**
prevents minting and issueing balance with over 100 bytes length

### CODE REVIEW

File: vm/systemSmartContracts/esdt.go [hunk 1]
```diff
@@ -539,6 +545,12 @@ func (e *esdt) mint(args *vmcommon.ContractCallInput) vmcommon.ReturnCode {
 	if returnCode != vmcommon.Ok {
 		return returnCode
 	}
+	if len(args.Arguments[1]) > core.MaxLenForESDTIssueMint {
+		returnMessage := fmt.Sprintf("max length for esdt issue is %d", core.MaxLenForESDTIssueMint)
```
Comment (bogdan-rosianu) [inline vm/systemSmartContracts/esdt.go:549]: `max length for esdt mint is %`

File: vm/systemSmartContracts/esdt_test.go [hunk 2]
```diff
@@ -167,6 +167,41 @@ func TestEsdt_ExecuteIssue(t *testing.T) {
 	assert.Equal(t, vmcommon.UserError, output)
 }
 
+func TestEsdt_ExecuteIssueToMuchSupply(t *testing.T) {
```
Comment (bogdan-rosianu) [inline vm/systemSmartContracts/esdt_test.go:170]: Too*

File: vm/systemSmartContracts/esdt.go (hunk 1 above)
Comment (sasurobert) [reply vm/systemSmartContracts/esdt.go]: done

File: vm/systemSmartContracts/esdt_test.go (hunk 2 above)
Comment (sasurobert) [reply vm/systemSmartContracts/esdt_test.go]: done

**Review by bogdan-rosianu — APPROVED** [review]

**Review by SebastianMarian — APPROVED** [review]

**Pull request merged**